.aegir.js

```diff
@@ -15,14 +17,28 @@ const before = (done) => {
   let count = 0
   const ready = () => ++count === 2 ? done() : null
 
-  sigServer.start({ port: 15555 }, (err, _server) => {
+  sigServer.start({
+    port: 15555
+  }, (err, _server) => {
     if (err) {
       throw err
     }
     server = _server
     ready()
   })
 
+  wsRendezvous.start({
+    port: 14444,
+    refreshPeerListIntervalMS: 1000,
+    strictMultiaddr: false
+  }, (err, _server) => {
+    if (err) {
+      throw err
+    }
+    server2 = _server
+    ready()
+  })
```

Comment:
@mkg20001 mind putting the two tasks, `sigServer.start` and `wsRendezvous.start` in a [`async.parallel`](http://caolan.github.io/async/docs.html#parallel) flow?

Reply:
They are already parallel...

Reply:
I mean, to make it more structured with Parallel. I also see Ready being called 3 times and only checked for 2